tedana/workflows/tedana.py

```diff
@@ -187,7 +196,9 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
                     tedort=False, gscontrol=None, tedpca='mle',
                     source_tes=-1, combmode='t2s', verbose=False, stabilize=False,
                     out_dir='.', fixed_seed=42, maxit=500, maxrestart=10,
-                    debug=False, quiet=False, png=False, png_cmap='coolwarm'):
+                    debug=False, quiet=False,
+                    png=False, png_cmap='coolwarm', user_tr=0.0
```

Comment:
I like `t_r` personally. I don't think we need the `user_`. I'd also prefer to default to `None` instead of 0.

Reply:
The `tr` from the data can be changed to something clearer, like `img_t_r` or something.